pages/language/index.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from 'next/link';
+import React from 'react';
+import Logo from "../../assets/icon/logo.svg";
+
+const index = () => {
+  return (
+    <header className="hidden md:block">
+      <nav className="flex flex-row items-center justify-between w-[90%] m-auto py-4 ">
+        <div className="flex flex-row items-center gap-9">
+          <div className="">
+            <img src={Logo.src} alt="" className="" />
```

Comment:
Rather than using the default html tag **<img/>** consider using the provided by next js called [Image](https://nextjs.org/docs/api-reference/next/image)